lab-ron/lib/read-files.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const fs = require('fs');
+
+module.exports = exports = {};
```

Comment:
@Rubiksron no need for this line as you are only exporting a single function - in this case, the export can be achieved by simply doing:

```
module.exports = function() { // do stuff here }
```